src/renderer/services/formatting.js

```diff
@@ -59,7 +59,7 @@ export default {
     return moment.unix(timestamp).format(formats.DATE_SHORT);
   },
 
-  formatMoney(value, symbol, defaultValue = 'N/A') {
+  formatMoney(value, symbol, defaultValue = 'N/A', abbreviate = false) {
```

Comment:
Don't need the default value since undefined evaluates as false; however explicit default is ok, so it's just a nitpick.